packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -7,6 +7,8 @@ import type {
   RawEnvironment,
 } from "../config";
 
+const isWindows = process.platform === "win32";
```

Comment:
pain...can you create an issue for normalizing paths across platforms? I think I wrote some logic for this in the webpack-plugin, it can just be moved or copied over to the `wrangler` tests

Reply:
Yes, there is a helper for this: https://github.com/petebacondarwin/wrangler2/blob/0a79d75e6aba11a3f0d5a7490f1b75c9f3e80ea8/packages/wrangler/src/__tests__/helpers/mock-console.ts#L28-L32

Reply:
Perhaps we need to move this somewhere that can be used from configuration.test.ts.

Reply:
A much simpler fix was to use path.posix to generate the path that we log to the terminal, since those paths will work on windows anyway.